.github/workflows/go.yml

```diff
@@ -22,3 +22,15 @@ jobs:
 
     - name: ci
       run: PATH=$(go env GOPATH)/bin:$PATH make ci
+
+    - name: integration
+      # skip running this action if the PR is coming from a fork:
+      if: github.event.pull_request.head.repo.full_name == github.repository
+      run: PATH=$(go env GOPATH)/bin:$PATH make integration
+      env:
+          AUTH0_CLI_CLIENT_NAME: ${{ secrets.AUTH0_CLI_CLIENT_NAME }}
+          AUTH0_CLI_CLIENT_DOMAIN: ${{ secrets.AUTH0_CLI_CLIENT_DOMAIN }}
+          AUTH0_CLI_CLIENT_ID: ${{ secrets.AUTH0_CLI_CLIENT_ID }}
+          AUTH0_CLI_CLIENT_SECRET: ${{ secrets.AUTH0_CLI_CLIENT_SECRET }}
+          AUTH0_CLI_REUSE_CONFIG: ${{ secrets.AUTH0_CLI_REUSE_CONFIG }}
```

Comment:
I'm not sure if we need `AUTH0_CLI_REUSE_CONFIG` on the secrets. It should be always false on CI if understood the code correctly; there's no config file to reuse here.

Reply:
what if we tweak the CLI codebase to look for all those ENV vars and skip the config file completely if the vars are present?

Reply:
nvm, we don't want to support client id/secret auth mechanism on the code base. In any case, it could look for an ENV var with the access token and skip config file, but that has nothing to do with this PR.